ckanext/harvest/logic/action/delete.py

```diff
@@ -5,30 +5,21 @@
 
 log = logging.getLogger(__name__)
 
+
 def harvest_source_delete(context, data_dict):
-    '''
-    Deletes an existing harvest source
+    '''Deletes an existing harvest source.
 
     This method just proxies the request to package_delete,
     which will delete the actual harvest type dataset and the
     HarvestSource object (via the after_delete extension point).
-
-    :param id: the name or id of the harvest source to delete
```

Comment:
I deleted the params because it doesn't accept an ID. I should have added `context` and `data_dict` as params.